resources/views/blocks/list.blade.php

```diff
@@ -1,7 +1,9 @@
 @php
 $tag = 'ul';
-if('ordered' === $data['type']){
-$tag = 'ol';
+if (isset($data['style']) && 'ordered' === $data['style']) {
```

Comment:
It could be more cleaner.

```
@php
$listType = $data['style'] ?? $data['type'] ?? 'unordered';

$tag = $listType === 'unordered' ? 'ul' : 'ol';
@endphp
```

Reply:
Thank you for reviewing the code. I have made the requested changes, and the code has been updated accordingly. Please take another look at the pull request, and let me know if everything now meets your expectations! :)